docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,128 @@
 # View
 
-The `View` class is the base class for all views. With this class you can manage the background color and images for UI components.
+The [View](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.BaseComponents.View.html) is the fundermental concept for the all components such as `Button`, `Image`, `Text`, etc.
+The `View` provides several [Properties](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.BaseComponents.View.html#properties), [Methods](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.BaseComponents.View.html#methods), and [Events](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.BaseComponents.View.html#events) which are commonly used on every components. To show the `View`, the `View` is needed to add on the [Window](https://developer.tizen.org/dev-guide/csapi/api/Tizen.NUI.Window.html) as child.
```

Comment:
Please use the links in https://samsung.github.io/TizenFX/latest/api/...

For example : 
 - https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#properties
 - https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#methods
...

Reply:
Modified all links to use "http://samsung.github.io"